apps/mobile/src/components/loading/fetch-wrapper.tsx

```diff
@@ -3,7 +3,7 @@ import { Error, FetchState, Loading } from '@/components/loading';
 import { assertUnreachable } from '@leather.io/utils';
 
 interface FetchWrapperProps {
-  data: FetchState<any>;
+  data: FetchState<unknown>;
```

Comment:
@camerow this was a comment from you [here](https://github.com/leather-io/mono/pull/1130#discussion_r2080400916)